cli/daemon/sqldb/docker/docker.go

```diff
@@ -200,6 +200,10 @@ func (d *Driver) clusterStatus(ctx context.Context, id sqldb.ClusterID) (status
 			if bytes.Contains(out, []byte("No such container")) {
 				continue
 			}
+			// Podman returns an empty array when a container is not found.
+			if bytes.Contains(out, []byte("[]")) {
```

Comment:
I'm a bit worried that checking for any string containing `[]` might lead to false negatives in certain situations. 

Does the podman output contain **only** `[]`? If so we could check something like `bytes.Equal(bytes.TrimSpace(out), []byte("[]"))`.

Reply:
Stdout will contain only `[]`, but stderr will contain the longer error text.

```sh
> podman container inspect foo > stdout 2> stderr
> cat stdout
[]
> cat stderr
Error: no such container "foo"
```

```sh
> podman image inspect foo > stdout 2> stderr
> cat stdout
[]
> cat stderr
Error: unable to inspect "foo": failed to find image foo: foo: image not known
```

This code path is currently using `CombinedOutput` to search across both stdout and stderr. If we want to use `bytes.Equal(bytes.TrimSpace(out), []byte("[]"))` we'd have to search stdout only. If we want to keep using `CombinedOutput` I can do the following:
- `bytes.Contains(out, []byte("no such container"))` for `docker container inspect`
- `bytes.Contains(out, []byte("failed to find image"))` for `docker image inspect`

What do you think?

Reply:
Makes sense. Yes, in that case checking for those strings make more sense I think.

Reply:
Done!